src/main/java/com/theironyard/Controllers/AuthenticationController.java

```diff
@@ -0,0 +1,61 @@
+package com.theironyard.Controllers;
+
+import com.theironyard.CommandObjects.UserCommand;
+import com.theironyard.Entities.User;
+import com.theironyard.Repositories.UserRepository;
+import com.theironyard.Utilities.PasswordHash;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.http.HttpSession;
+
+/**
+ * Created by darionmoore on 1/14/17.
+ */
+@Controller
+public class AuthenticationController {
+    public static final String CURRENT_USER = "userName";
+
+    @Autowired
+    UserRepository userRepository;
+
+
+    /*
+    Gets the login page
+     */
+    @RequestMapping(path = "/login", method = RequestMethod.GET)
+    public String loginPage(HttpSession session){
+        if(session.getAttribute(CURRENT_USER) == null){
```

Comment:
This if statement doesn't do anything at all.  If you return the same thing in every path through the code then that is a big red flag